### PR TITLE
Added the dnxtools feed to dotnet restore

### PR DESCRIPTION
**Pull request by ajaybhargavb**
@pranavkm / @victorhurdugaci / @ryanbrandenburg

### CODE REVIEW

**Comment by pranavkm** [comment]
:shipit:

File: build/KoreBuild.ps1 [hunk 1]
```diff
@@ -45,7 +45,7 @@ if (!($env:Path.Split(';') -icontains $dotnetLocalInstallFolderBin))
 if (!(Test-Path "$koreBuildFolder\Sake")) 
 {
     $toolsProject = "$koreBuildFolder\project.json"
-    &dotnet restore "$toolsProject" --packages "$koreBuildFolder" -v Minimal
+    &dotnet restore "$toolsProject" --packages "$koreBuildFolder" -f https://www.myget.org/F/dnxtools/api/v3/index.json -v Minimal
```
Comment (victorhurdugaci) [inline build/KoreBuild.ps1:48]: what? Why do you need this, there's already a nuget config in this folder

**Comment by victorhurdugaci** [comment]
You don't need this change

**Comment by ajaybhargavb** [comment]
This change was to get the build to pass. Filed an issue to investigate this.

**Pull request merged**